### PR TITLE
install gem 'devise', create model 'user', add mailcatcher integration

### PR DESCRIPTION
**Pull request by HannaKukhareva95**
install gem 'devise'
create model 'user'
add integration with 'mailcatcher' by adding configurations to 'config/environments/development.rb'
add gem 'omniauth' and 'omniauth-facebook' for facebook authentication
add email confirmation settings

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class HomeController < ActionController::Base
```
Comment (ilyabyar) [inline app/controllers/home_controller.rb]: Do we need that controller?

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  devise_for :users, controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
+  root 'home#index'
```
Comment (ilyabyar) [inline config/routes.rb]: Let's remove that code, along with  controller

File: Gemfile [hunk 3]
```diff
@@ -3,52 +3,33 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.3'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```
Comment (ilyabyar) [inline Gemfile]: I am just curious, was there any reason to make those changes in a Gemfile?

Comment (HannaKukhareva95) [reply Gemfile]: for me, gemfile is more readable when it does not contain these comments

**Pull request merged**